### PR TITLE
Scan: Remove redundant loading component

### LINKED ISSUE #1556: Remove redundant loading component
Currently, this loading (image is below) is implemented in separate files. Let's make a component for this and pass width or height as props.

image

### PR DESCRIPTION
**Pull request by Pitchanai**
Fixed: #1556 

## Implementation details
- Remove redundant loading component

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [ ] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: scan/src/components/data-source/DataSourceExecute.re [hunk 1]
```diff
@@ -107,9 +107,7 @@ type result_t =
   | Success(result_data_t);
 
 let loadingRender = (wDiv, wImg, h) => {
-  <div className={Styles.withWH(wDiv, h)}>
-    <img src=Images.loadingCircles className={Styles.withWH(wImg, h)} />
-  </div>;
+  <div className={Styles.withWH(wDiv, h)}> <Loading width=wImg height=h /> </div>;
```
Comment (evilpeach) [inline scan/src/components/data-source/DataSourceExecute.re]: Can we remove `<div>`?

File: scan/src/components/Loading.re [hunk 2]
```diff
@@ -0,0 +1,21 @@
+module Styles = {
+  open Css;
+
+  let loading = (w, h, mb) =>
+    style([
+      width(w),
+      height(h),
```
Comment (evilpeach) [inline scan/src/components/Loading.re]: IMO, we can use only the width. The height will be adjusted automatically.

**Review by evilpeach — CHANGES_REQUESTED** [review]
Please fix the conflict.

File: CHANGELOG_UNRELEASED.md [hunk 3]
```diff
@@ -22,6 +22,36 @@
...
+- (impv) [\#2676](https://github.com/bandprotocol/bandchain/pull/2676) Setup cypress to scan, added new sendToken testcase
+- (bugs) [\#2673](https://github.com/bandprotocol/bandchain/pull/2673) Fixed sorting on moniker with emoji
+- (impv) [\#2672](https://github.com/bandprotocol/bandchain/pull/2672) Hid the proposal desc from tx index page table
+- (impv) [\#2671](https://github.com/bandprotocol/bandchain/pull/2671) Adjusted tooltip width and added webapi
+- (impv) [\#2670](https://github.com/bandprotocol/bandchain/pull/2670) Format the count number
+- (feat) [\#2669](https://github.com/bandprotocol/bandchain/pull/2669) Add active validator's rank on Validator Index Page
+- (bugs) [\#2666](https://github.com/bandprotocol/bandchain/pull/2666) Handle rate limit msg when decoding
+- (impv) [\#2663](https://github.com/bandprotocol/bandchain/pull/2663) Fixed and updated `ChainIDBadge`
+- (impv) [\#2644](https://github.com/bandprotocol/bandchain/pull/2644) Added `netlify.toml` configuration
+- (feat) [\#2594](https://github.com/bandprotocol/bandchain/pull/2594) Added meta og tag to scan
```
Comment (pzshine) [inline CHANGELOG_UNRELEASED.md]: Remove all except #2790

**Review by pzshine — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]

**Review by pzshine — APPROVED** [review]

**Pull request merged**